### PR TITLE
Bug fix 3.5/fix internal issue 4451

### PR DESCRIPTION
**Pull request by hkernbach**
### Scope & Purpose

This PR re-enables Dump & Restore suite
- Fixed failed Foxx test.
- Fixed restore of users. 

Backports for 3.4 / 3.6 (devel) are required. 
- [x] The behavior in this PR can be (and was) *manually tested* (support / qa / customers can test it)
Yes, execute `./scripts/unittest dump_authentication --cluster true`

- [x] The behavior change can be verified via automatic tests
Yes, they will run in Jenkins again. 

#### Related Information

*(Please reference tickets / specification etc )*

- [x] There is an internal planning ticket:  #4451
- [x] There is a *ES-383* (In case a customer was affected / involved): 

### Testing & Verification
This change is already covered by existing tests, such as *(please describe tests)*.
- [x] dump & restore suite

### CODE REVIEW

**Comment by hkernbach** [comment]
Jenkins: http://jenkins01:8080/view/PR/job/arangodb-matrix-pr/7380/

File: js/client/modules/@arangodb/testsuites/dump.js [hunk 1]
```diff
@@ -410,20 +453,6 @@ function dumpMultiple (options) {
 }
 
 function dumpAuthentication (options) {
-  if (options.cluster) {
```
Comment (mchacki) [inline js/client/modules/@arangodb/testsuites/dump.js]: yay \o/

**Comment by hkernbach** [comment]
Next try: http://jenkins01:8080/view/PR/job/arangodb-matrix-pr/7405/

**Comment by KVS85** [comment]
Tests blue!

**Pull request merged**